bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -231,7 +231,9 @@ spec:
                 {{- end }}
             {{- end }}
             {{- if and .Values.ingress.enabled .Values.ingress.hostname }}
-            - name: KC_HOSTNAME_URL
+            - name: KEYCLOAK_HOSTNAME_STRICT
+              value: "true"
```

Comment:
I'm on the fence whether this should be automatically enabled. It is more secure, but may be not valid in some cases.

Reply:
According to the [docs](https://www.keycloak.org/server/hostname):
> Should always be set to true in production, unless your reverse proxy overwrites the Host header. If enabled, the hostname option needs to be specified.

Perhaps its value is based on the `production` chart parameter.

Reply:
What about parameterizing it?

Reply:
added the parameter with default as `false` so its not a breaking change